src/client/components/Delete_Comfim/Delete_Comfirm.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ButtonComponent from './Button.component';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, select, boolean } from '@storybook/addon-knobs';
```

Comment:
I think you don't need knobs here since there's no variable parts in this component...